clouds/snowflake/modules/doc/h3/H3_POLYFILL.md

```diff
@@ -1,15 +1,21 @@
 ## H3_POLYFILL
 
 ```sql:signature
-H3_POLYFILL(geography, resolution)
+H3_POLYFILL(geography, resolution)  
+H3_POLYFILL(geography, resolution, mode)
 ```
 
 **Description**
 
-Returns an array with all the H3 cell indexes **with centers** contained in a given polygon. It will return `null` on error (invalid geography type or resolution out of bounds).
+Returns an array with all H3 cell indicies contained in the given polygon. There are three modes which decide if a H3 cell is contained in the polygon:  
 
-* `geography`: `GEOGRAPHY` **polygon** or **multipolygon** representing the shape to cover.
-* `resolution`: `INT` number between 0 and 15 with the [H3 resolution](https://h3geo.org/docs/core-library/restable).
+- **center** (Default) - The center point of the H3 cell must be within the polygon
```

Comment:
I would look at the formatting and docs of:
core/clouds/bigquery/modules/doc/h3/H3_POLYFILL_MODE.md

The text doesn't necessarily have to be equal as each provider is different but maybe would keep the format.

Reply:
Done. I also copied the intersects/contains/center mode images which i notice all the other polyfill docs use